src/lsptoolshost/roslynProtocol.ts

```diff
@@ -100,3 +108,9 @@ export namespace ProjectInitializationCompleteNotification {
     export const messageDirection: MessageDirection = MessageDirection.serverToClient;
     export const type = new NotificationType(method);
 }
+
+export namespace SourceGeneratorGetTextRequest {
+    export const method = 'sourceGenerator/getText';
```

Comment:
this is roslyn-specific right?  curious if it might make sense to add some sort of roslyn_ prefix on things specific to us.

this is a non-blocking question for this PR.  

@dibarbet for his thoughts too.

Reply:
Agree, we should prefix roslyn methods.  Maybe similar to the VS protocol does `_vs_` we could do `_roslyn_`

Reply:
@dibarbet Should I start with this one then and we go back to the other ones later?

Reply:
Yeah lets start here, we can fix other ones in separate PRs

Reply:
love it. :)

Reply:
Rebased and this is now called sourceGeneratedFile/_roslyn_getText, which matches how we're naming other things.